src/Momento.Sdk/ITopicClient.cs

```diff
@@ -44,6 +44,7 @@ public interface ITopicClient : IDisposable
     /// <param name="cacheName">Name of the cache containing the topic.</param>
     /// <param name="topicName">Name of the topic.</param>
     /// <param name="resumeAtSequenceNumber">The sequence number of the last message.
+    /// <param name="resumeAtSequencePage">The sequence page of the last message.
```

Comment:
github complaint about a missing `</param>`

Reply:
Fixed in [4953482](https://github.com/momentohq/client-sdk-dotnet/pull/588/commits/4953482a0839d9dc44273a73e7552baea5f07873)